Makefile

```diff
@@ -169,22 +188,31 @@ endif
 ## Build site. This target should only be used by Netlify and Prow
 build: envvar
 	@echo "${GREEN}Makefile: Build mkdocs site${RESET}"
-	which python3.7
-	python3.7 -m venv /tmp/venv
+	which $(PYTHON)
+	$(PYTHON) -m venv /tmp/venv
 	. /tmp/venv/bin/activate
-	pip3 install mkdocs mkdocs-awesome-pages-plugin mkdocs-htmlproofer-plugin
+	$(PIP) install mkdocs mkdocs-awesome-pages-plugin mkdocs-htmlproofer-plugin
 	@echo
 	@echo '*** BEGIN cat mkdocs.yml ***'
 	@cat mkdocs.yml
 	@echo '*** END cat mkdocs.yml ***'
 	mkdocs build -f mkdocs.yml -d site
 
+
 ## Run site.  App available @ http://0.0.0.0:8000
 run: | envvar stop
 	@echo "${GREEN}Makefile: Run site${RESET}"
-	${CONTAINER_ENGINE} run -d --name userguide --net=host -v ./:/srv:ro${SELINUX_ENABLED} --mount type=tmpfs,destination=/srv/site kubevirt-userguide:latest /bin/bash -c "mkdocs build -f /srv/mkdocs.yml && mkdocs serve -f /srv/mkdocs.yml -a 0.0.0.0:8000"
+	${CONTAINER_ENGINE} run \
+        -d \
+        --name userguide \
+        -p ${LOCAL_SERVER_PORT}:8000 \
+        -v ${PWD}:/srv:ro${SELINUX_ENABLED} \
+        --mount type=tmpfs,destination=/srv/site \
+        kubevirt-userguide:latest \
+        /bin/bash -c "mkdocs build -f /srv/mkdocs.yml && mkdocs serve -f /srv/mkdocs.yml -a 0.0.0.0:8000"
+	@echo
+	@echo "${AQUA}Makefile: Server now running at [https://localhost:$(LOCAL_SERVER_PORT)]${RESET}"
 	@echo
-
```

Comment:
Let's keep twp spaces between targets.  That's just my personal preference.

Reply:
Can do, this was unintentional on my part

- [ ] Fix two space gap on 187